charts/fluentd/templates/_helpers.tpl

```diff
@@ -35,6 +35,8 @@ Builds the full logzio listener host
 {{- define "logzio.listenerHost" }}
 {{- if or ( eq $.Values.secrets.logzioListener "listener.logz.io" ) ( eq $.Values.secrets.logzioListener " " ) -}}
 {{- printf "https://listener.logz.io:8071" }}
+{{- else if ne $.Values.secrets.customEndpoint "" -}}
+{{- printf "%s" .Values.secrets.customEndpoint  }}
```

Comment:
Can we first have custom endpoint, and after that listener? so that customEndpoint has precedence and overrides logzioListener.